src/components/ButtonPanel.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable class-methods-use-this */
+import React from 'react';
+import '../index.scss';
+import Button from './Button';
+
+export default class ButtonPanel extends React.Component {
+  renderButton(name) {
+    const wide = name === '0' ? '50%' : '25%';
+    const color = !['÷', 'X', '-', '+', '='].includes(name)
+      ? 'light-grey'
+      : 'orange';
+    return <Button name={name} wide={wide} color={color} />;
+  }
```

Comment:
meaning you'll tweak this `if` statement a little regarding `wide` property which should be boolean

Reply:
Thanks for pointing it out! Is it solved!